tutorials/schedule-dataflow-jobs-with-cloud-scheduler/index.md

```diff
@@ -67,7 +70,54 @@ EOT
 }
 ```
 
-Afterwards you are all set up! 
+## Instructions
+
+The following step-by-step instructions can be used to create a sample Dataflow pipeline with Cloudbuild.
+
+First, open Cloud Shell and clone the repository.
+
+```
+git clone https://github.com/GoogleCloudPlatform/community
+cd community/tutorials/schedule-dataflow-jobs-with-cloud-scheduler/scheduler-dataflow-demo/
+```
+
+Create a bucket on Google Cloud Storage, which will be used to store terraform states and Dataflow templates.
+Replace BUCKET_NAME and PROJECT_ID with your own choice.
+You can skip this step if you already have one GCS bucket created.
+
+```
+export BUCKET_NAME=[BUCKET_NAME]
+export BUCKET=gs://${BUCKET_NAME}
+gsutil mb -p [PROJECT_ID] ${BUCKET}
+```
+
+The next step is to create a backend for Terraform to store the states of 
+GCP resources. Run the below command to create a remote backend service using GCS as the storage.
+```
+cd terraform
+cat > backend.tf << EOF
+terraform {
+ backend "gcs" {
+   bucket  = "${BUCKET_NAME}"
+   prefix  = "terraform/state"
+ }
+}
+EOF
+```
+
+Follow the [instruction](https://cloud.google.com/scheduler/docs/quickstart) to create a App Engine, which is needed to
+set up Cloud Scheduler jobs.
+
+Note: Cloud Scheduler jobs need to be created in the same region as the App engine. 
+
+Afterwards, you can submit a Cloudbuild job to create all the resources.
+Replace the *REGION* and *PROJECT_ID* with your own values.
```

Comment:
Use ${REGION}, ${GOOGLE_CLOUD_PROJECT} here to avoid manual substitution

Reply:
done.